Paddlepaddle support

bstrzele opened the pull request.

bstrzele committed:
added support for paddlepaddle format

bstrzele committed:
added support for paddlepaddle format

bstrzele committed:
added segmentation demo

bstrzele committed:
added segmentation demo

dtrawins commented on demos/segmentation_using_paddlepaddle_model/python/requirements.txt:
```diff
@@ -0,0 +1,3 @@
+ovmsclient
+paddlepaddle
```
I don't see it used on the example client

atobiszei committed:
Remove TODO's (#1306)

The only remainings are KFS binary inputs related TODO's which should
come soon.

* Fix includes & forward declares
* Move tfs related utils outside TensorInfo to decouple it from TFS

JIRA:CVS-84365

bstrzele committed:
resolved convention

bstrzele committed:
resolved convention

atobiszei committed:
Fix ONNX functional tests after change in ONNX repo (#1316)

This change makes models support dynamic batch size:
https://github.com/onnx/models/commit/652f4e4af7975c8e7a505c4b6e0f8ac72d8260ea

We need to hardcode ONNX Model version. We can bump model version later
with tests refactor. This would require fix on OV/ONNX side as model
reports that dynamic bs model have dynamic shape only on input not
output.

JIRA:CVS-87169

michalkulakowski committed:
Mkulakow/kfs binary inputs client (#1309)

* CVS-86784 KFS binary inputs client + minor fixes

dtrawins committed:
update OV package to 4th july (#1314)

bstrzele committed:
added overview

bstrzele committed:
added overview

bstrzele committed:
Fixed typo

Co-authored-by: Miłosz Żeglarski <milosz.zeglarski@intel.com>

bstrzele committed:
Fixed typo

Co-authored-by: Miłosz Żeglarski <milosz.zeglarski@intel.com>

bstrzele committed:
Added dependency links

Co-authored-by: Miłosz Żeglarski <milosz.zeglarski@intel.com>

bstrzele committed:
Added dependency links

Co-authored-by: Miłosz Żeglarski <milosz.zeglarski@intel.com>

bstrzele committed:
Merge branch 'CVS-86567-paddlepaddle-support' of https://github.com/openvinotoolkit/model_server into CVS-86567-paddlepaddle-support
